kernel/src/scan/log_replay.rs

```diff
@@ -172,7 +170,7 @@ impl AddRemoveDedupVisitor<'_> {
                             "missing partition value for field index {field_idx}"
                         )));
                     };
-                    Ok(partition_value.into())
+                    Ok(Arc::new(partition_value.into()))
```

Comment:
Just for my understanding: This is where the partition value cache that @scovich  mentioned would live right?

Reply:
see next PR :) #1172